apps/website/build-scripts/video-loader.cjs

```diff
@@ -266,26 +343,77 @@ const videoLoader = async (context, content) => {
   // Based on https://github.com/vercel/next.js/blob/888384c5e853ee5f9988b74b9085f1d6f80157a3/packages/next/src/build/webpack/loaders/next-image-loader/index.ts#L66-L74
   // We don't emit for the server as videos are considered traceable and this breaks things (see https://github.com/vercel/next.js/pull/41554/files)
   if (!options.isServer) {
-    files.forEach(({ name, content }) => context.emitFile(name, content, null));
+    files.forEach(
+      ({ name, content }) => content && context.emitFile(name, content)
+    );
   }
 
+  // Collect stats on cache/skipped files
+  const stats = files.reduce(
+    (acc, { cached, skipped }) => ({
+      cached: acc.cached + (cached ? 1 : 0),
+      skipped: acc.skipped + (skipped ? 1 : 0),
+      total: acc.total + 1,
+    }),
+    { cached: 0, skipped: 0, total: 0 }
+  );
+
   // Return the object with the paths
-  return `export default ${JSON.stringify(obj)};`;
+  return {
+    output: `export default ${JSON.stringify(obj)};`,
+    stats,
+  };
 };
 
+/**
+ * Webpack hook to clean up old cache files
+ *
+ * @return {Promise<void>}
+ */
+const cacheCleanup = async () => {
+  // Get all files in the cache dir
```

Comment:
this should handle the case if the cacheDir does not exist yet. that might happen when doing a clean `pnpm build`.
Something like `if (!fs.existsSync(cacheDir)) return;`

Reply:
🤔 It should never be possible for this directory to not exist -- the loader always runs before this, and always writes to the cache dir?

Reply:
I guess I should handle the case that something manages to remove that directory between the loader running and the cleanup plugin running 👍

Reply:
It happened to me. I 'rm -rf'ed .next and ran 'pnpm build'

Reply:
Oh right yes, the server build happens first that doesn't emit assets 🤦